### PR TITLE
Lineintegral

### PR DESCRIPTION
**Pull request by abh2k**
* vector
  *  Added line integral in vector module

### CODE REVIEW

**Review by friyaz — CHANGES_REQUESTED** [review]
There are spelling mistakes.
feild -> field
staarting ->starting.

**Review by friyaz — COMMENTED** [review]
I think the API can be improved.
Instead of passing parametric equation of curve, we should make a class to represent curve and then pass object of this class to `LineIntegrate`.

Even if we go with this API, user should be able to pass parametric equation of line as a tuple rather than a single term.

**Review by friyaz — COMMENTED** [review]
This implementation requires field to be vector field. It does not work for scalar field.

**Comment by jksuom** [comment]
There is `line_integrate` in `integrals`. There should preferably be only one function for all line integrals.

**Comment by abhinav28071999** [comment]
@jksuom  Hey, i saw the idea of vector integration on the ideas page of Gsoc. So i was hoping to 
make a patch regarding that. It was given to implement line integral , surface integral and volume integral. So i thought to start line integration. Please tell me what are the thing required in the vector integration idea page.

**Comment by jksuom** [comment]
The vector integration idea has been discussed several times on the mailing list, for instance, https://groups.google.com/forum/#!msg/sympy/lBi1VvkYPaQ/isjPRi0fAgAJ and https://groups.google.com/forum/#!msg/sympy/9GFdlK3btwo/BJR390oXBgAJ.

File: sympy/vector/functions.py [hunk 1]
```diff
@@ -517,3 +517,43 @@ def orthogonalize(*vlist, **kwargs):
         ortho_vlist = [vec.normalize() for vec in ortho_vlist]
 
     return ortho_vlist
+
+def lineintegral(l,field,coord_sys, start,end ):
```
Comment (smichr) [inline sympy/vector/functions.py]: spaces after commas

File: sympy/vector/functions.py [hunk 2]
```diff
@@ -517,3 +517,43 @@ def orthogonalize(*vlist, **kwargs):
...
+    """
+    Computes line integral.A line integral is an integral where the function
```
Comment (smichr) [inline sympy/vector/functions.py]: space after period.

remove double period, etc...

newline before Parameters

File: sympy/vector/functions.py (hunk 1 above)
Comment (smichr) [inline sympy/vector/functions.py]: Is it possible to infer the coordinate system from the equation given?

**Comment by czgdp1807** [comment]
Closing since these comments, https://github.com/sympy/sympy/pull/16558#issuecomment-480245858 and https://github.com/sympy/sympy/pull/16558#issuecomment-480215491 suggest that there is already an existing function satisfying the requirements. Thanks for the contributions. Please let me know if it is needed to be re-opened.

**Pull request closed without merging**